### ISSUE TITLE
[request] Show stat distribution in power up menu

### ISSUE THREAD

**Issue by udnp** [farkam135/GoIV#1022]
User request
https://www.reddit.com/r/GoIV/comments/a3wc8n/request_show_stat_distribution_in_power_up_menu/

My private custom build for GoIV v4.3.3
![device-2018-12-07-150002](https://user-images.githubusercontent.com/17121615/49630500-c75e7d00-fa31-11e8-8f67-4d3fedf40df6.png)

It should be better to be adapted to v5 UI.

**Comment by AbcentMineDead** [comment]
Curious if this implementation is just showing base stats + IV or if it has the potential to be more detailed.

This is an image I grabbed to show an example of what I'm talking about. http://imgur.com/a/o5YmNgV

Note: I don't know why or where these numbers come from exactly but maybe you'll steer me clear.
http://imgur.com/a/o5YmNgV

**Comment by udnp** [comment]
> Curious if this implementation is just showing base stats + IV or if it has the potential to be more detailed?

My picture is just on base stats not including IV.
and I think so for this users request, currently.

> This is an image I grabbed to show an example of what I'm talking about. http://imgur.com/a/o5YmNgV

Are your pictures from Calcy IV app?

**Comment by AbcentMineDead** [comment]
Yes my image is from calcy IV. I apologise but I'm very interested in why the "base stats" have such a disparity that I don't understand from the calcy IV power up page.

Is a L1 Bulbasaur's base stats:
ATK 118 + IV
DEF 111 + IV
STA 128 + IV

Is a L40 Bulbasaur's base stats:
ATK 118 + IV
DEF 111 + IV
STA 128 + IV

Or do they increase in some way I don't understand as they are powered up? It seems they do. Sorry I'm just genuinely curious.

Cheers!! I only use GoIV for my scans but I do like to peek around the other options. I've used GoIV since the author publicly made it available. =)

<3 <3 <3

**Comment by AbcentMineDead** [comment]
Something called CP mulitplier or CP*ML. I'm just trying to help.

**Comment by udnp** [comment]
> Is a L1 Bulbasaur's base stats:
> ATK 118 + IV
> DEF 111 + IV
> STA 128 + IV
> 
> Is a L40 Bulbasaur's base stats:
> ATK 118 + IV
> DEF 111 + IV
> STA 128 + IV

My understanding is the same.
When a pokemon becomes level up, its ability of attack, defense, and stamina increase.
But its base stats do not change, I think.

> Or do they increase in some way I don't understand as they are powered up? It seems they do. Sorry I'm just genuinely curious.

They may know the logic to add a pokemon's increased abilities to its stats and to show them.
If we could know that logic, we would have possibility to implement it to GoIV...

**Comment by AbcentMineDead** [comment]
Well that explains that! I'll dig around and see if I can find that logic publicly listed. To be blunt I'd love the base stats to show just as you showed. I enjoy doing a little bit of guessing and studying with out a certainty. If I really really need to know how/what a Pokemon will be doing I export to PB and sim.

I'm newer to GitHub and how to use it correctly. How do I obtain your version and is it up to date with stats but not UI? Sorry if I'm understanding wrong.

**Comment by udnp** [comment]
thanks for your interest in my impl.
i have not pushed that to github my GoIV🇯🇵 repo yet.

sorry today is a Japaese party 忘年会 for Ingress players.
so next day i will publish my patch for this impl.
